### DISCUSSION TITLE
The shape of the model can't be displayed

### DISCUSSION THREAD

**Discussion by myl-Giser** [iTwin/itwinjs-core#5416] [Q&A]
I have a dgn file containing four models, two of which are mesh and two of which are shape, as follows:
https://user-images.githubusercontent.com/81339378/232998179-cbead6c4-7016-4a03-807d-ecc87f9a9434.png
But I used Synchronizer to sync itwin, and when I opened it, everything was displayed as a mesh, and only one part of the model was displayed as a shape,as follows:
https://user-images.githubusercontent.com/81339378/233001376-2b3a6759-b7d6-4671-a8d0-d9c92c70f26f.png
Why is this and how can it be fixed?
This is my dgn file.
[models.zip](https://github.com/iTwin/itwinjs-core/files/11270317/models.zip)

**Comment by pratikshan85** [comment]
@myl-Giser We are checking with our internal dev team to solve the issue.

**Reply by myl-Giser** [reply]
ok,thanks@pratikshan85

**Reply by pratikshan85** [reply]
@myl-Giser Can you please provide your ProjectId on which you are creating in your iModel ? We need the projectId to fix the above issue .

**Reply by myl-Giser** [reply]
projectId=9d564918-e425-4d59-8062-e8df32c753a5
iModelId=c7726e19-7db7-4a0e-a80f-fad24b0d227d
ChangeSetId=41587a834f9e7006ad3c919fe59ea75527025f67

**Reply by pratikshan85** [reply]
@myl-Giser Can you please check now , whether it is working as expected or not ? 
Please use the same project.

**Reply by myl-Giser** [reply]
@pratikshan85 Thank you for your feedback, but it's still the same as before

**Reply by pratikshan85** [reply]
@myl-Giser Have you rerun the Synchronizer once again?
If not please rerun the Synchronizer for that particular project id and check once again.

**Reply by myl-Giser** [reply]
@pratikshan85 I rerun the Synchronizer, and it still doesn't change.
My iTwin Synchronizer version is 1.1.353,Does it matter？

**Reply by pratikshan85** [reply]
@myl-Giser Can you please try running the synchronizer through cloud service and check?
Please update your Synchronizer version to 

![MicrosoftTeams-image (11)](https://user-images.githubusercontent.com/48123387/234482622-f1847378-cae3-42d8-93da-584d3260623e.png)
 Then run your Synchronizer to check the result.

**Reply by myl-Giser** [reply]
hi,pratikshan85.
 Do I need to update Synchronizer or update MicroStation?

**Reply by pratikshan85** [reply]
@myl-Giser Update Synchronizer.

**Comment by myl-Giser** [comment]
hi,pratikshan85,I ran the synchronizer through the cloud service and the results were correct,as follows:
https://user-images.githubusercontent.com/81339378/234520628-59d4e293-2c80-4574-bd67-93ff68bd90f7.png
But I used itwin Synchronizer client to synchronize the result is still wrong,
My Synchronizer version  has been updated, as follows:
https://user-images.githubusercontent.com/81339378/234522025-7ab669b6-f9f6-4522-b9ad-edd534cd824f.png

**Reply by pratikshan85** [reply]
@myl-Giser Have you used the same project & reran the synchronizer after updating the connector?

**Reply by myl-Giser** [reply]
Yes,I used the same project.

**Reply by myl-Giser** [reply]
@pratikshan85 Thans for you reply,Now the results are correct.
I want to know what you has been modified,What version of itwin.js do you use.
Thanks！

**Reply by myl-Giser** [reply]
@pratikshan85  I want to know what causes this problem，Is it related to the version of itwin.js?Which version of itwin.js are you using？

**Reply by pratikshan85** [reply]
@myl-Giser  The issue is not about the iTwin.js version. If you run a connector then this is the default behavior that you are getting. To get the expected behavior  (as you wanted) internally we need to set a flag in the connector and that is what we did for now.

**Reply by myl-Giser** [reply]
@pratikshan85 How to set the flag in the connector？
I used Get started on the official website to create the imodel and uploaded file it from the local，but it was wrong.

**Reply by pratikshan85** [reply]
@myl-Giser By default the behavior of the connector is what you are getting (everything was displayed as a mesh, and only one part of the model was displayed as a shape).To get your desired result you need to request us and we will internally update the flag based on ProjectId.
